solara/components/file_drop.vue

```diff
@@ -1,6 +1,13 @@
 <template>
   <div ref="dropzone" class="solara-file-drop" effectAllowed="move">
-    {{ (file_info && file_info.length === 1) ? file_info[0].name : label }}
+    <div v-if="file_info && file_info.length">
```

Comment:
Ideally we would also keep the HTML the same for the single file case, as users might have CSS rules that depend on that structure. You can use `<template v-if=`, this wil not produce an HTML element in the output.

Reply:
I guess, this one will solve the problem.

~~~vue
<template>
  <div ref="dropzone" class="solara-file-drop" effectAllowed="move">
    <template v-if="file_info && file_info.length === 1"> 
      {{ file_info[0].name }}
    </template>
    <template v-else-if="file_info && file_info.length > 1"> 
      <div v-for="file in file_info">
        {{ file.name }}
      </div>
    </template>
    <template v-else> 
      {{ label }}
    </template>
  </div>
</template>
~~~